builder/utils.go

```diff
@@ -117,3 +122,21 @@ func SendHTTPRequest(ctx context.Context, client http.Client, method, url string
 
 	return resp.StatusCode, nil
 }
+
+func publicKeyFromHex(pubKeyHex string) (phase0.BLSPubKey, error) {
```

Comment:
This is BLS public key to phase0-BLS public key, what ever this "phase0" is.
Please add a comment to not be confused with "old" signing keys.